data/hai/hai reveal 1 intro.txt

```diff
@@ -649,18 +648,25 @@ mission "Hai Leaks Response 2C"
-		log "People" "Terry Adrianopoulos" `Protege to the Syndicate CEO, Terry Adrianopoulos has been nominated by her mentor Alastair Korban to represent the Syndicate's interests in the crisis of the Hai's secrecy. She is a very capable woman who appears well equipped for climbing the corporate ladder.`
+		log "People" "Terry Adrianopoulos" `Terry Adrianopoulos is the secretary of Samuel Remington, or at least that's how he puts it. Remington mentioned an internal political situation within the Syndicate while glancing at Terry, so there may be something larger at play with her presence. Hopefully it doesn't conflict with resolving the situation with the Hai.`
 		conversation
-			`Approaching Foundry, you are hailed on a private encrypted channel and directed to a private hangar where you are greeted personally by Syndicate CEO Alastair Korban.`
-			`	He shakes your hand and smiles warmly. "Apologies, Captain <last>, do excuse the cloak and dagger, but in the current media climate you and I are both too well-known to be seen meeting without causing the rumor mill to grind out a new story."`
-			`	"Now, allow me to introduce you to my protege, Terry Adrianopolous. She will be accompanying you as the Syndicate representative."`
-			`	"Pleasure to meet you," she says with a broad smile. The young woman goes from calculating to gregarious in the blink of an eye as your attention shifts to her. She is pleasant and approachable, and very articulate, but from your brief glimpse beforehand you have no doubt she's cut from the sort of cloth that makes her future CEO material. You are surprised that a woman would have made it this far in the Syndicate, but if she has gotten this far, she must be very skilled at what she does.`
-			`	"By the way," says Alastair, pausing briefly as he turns to leave, "The Navy and my security forces are working jointly to get some of our message relays up in the Far North to link through Ultima Thule. The pirates are wont to destroy them whenever they find them, but we should be able to rely on at least basic communication to get through if you need it, rather than flying back and forth for literally every question. That's all."`
-			branch delayedretire
-				not "free worlds reconciliation"
-			`	He leans in close to you and says quietly, "I know I said I was retiring, but Terry convinced me to clean up my mess first. I am atoning for my failures. Do look after her; she's a very loyal colleague."`
-			label delayedretire
-			`	Alastair says goodbye and departs, leaving you with Terry.`
+			`Approaching Foundry, you are hailed and directed to a private hangar where you are greeted personally by Samuel Remington. His appearance is rather casual compared to what you would expect, wearing khakis and a collared tee as opposed to the typical suit and tie business atire that you see on Syndicate businessmen. It is certainly in contrast to a young woman standing behind him, who is wearing the usual business attire in this region.`
```

Comment:
While Remington is in business casual right now, that isn't what he's wearing when he's introduced in the FW campaign. It's not spelled out or anything, but the difference in dress is because to him, this isn't really a business situation and he's taking a rather relaxed attitude to the events, which is in contrast to Terry who is all business with what's going on.